scripts/Jenkinsfile-deploy

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/groovy
+
+/*
+ * This file is part of the Software Factory project
+ * Copyright (C) Pelagicore AB
+ * SPDX-License_identifier: LGPL-2.1
+ * This file is subject to the terms of the LGPL-2.1 license.
+ * Please see the LICENSE file for details.
+ */
+
+//
+// Please note, this will only work on the Jenkins set up used for pelux.io
+// The only reasons for it being in this repo are 1) it has to do with the PELUX
+// Software Factory, and 2) it keeps the script under version control rather
+// than on the Jenkins instance.
+//
+// In short, this script looks for all tags on the remote, and builds the SWF
+// for them, plus master. If there has been changes since last successful build,
+// we build all tags and branches, and deploy them to the web server
+//
+
+// We run this on any node that has Sphinx and web server access
+node('Sphinx && WWW') {
+
+    // For scope reasons we define it here
+    def buildList = null
+
+    // We copy everything to a staging directory to avoid 404 while the job is running
+    String stagingDirectory = "/var/www/software-factory-staging/"
+    String swfDirectory = '/var/www/software-factory/'
```

Comment:
Those should be environment variables with a default if not set, so one can chose the path instad the path being hard coded. I would for example want to have the staging dir not accessible through Nginx and HTTP, but instead in `/tmp` for example.

Reply:
I agree on having the directories as env vars with defaults. Setting the staging dir to /tmp could work but if /tmp and /var/www (or whatever) are on different disks this would increase the potential downtime a lot. Not a problem right now, sure, but still.

Reply:
I just meant that if I'm running it on my own infrastructure I should be able to change the directories, it was just an example why I wanted to have it as env-variables.